docs/api.md

```diff
@@ -79,10 +80,9 @@ This class exposes the [Vantiq REST API](https://dev.vantiq.com/docs/system/api/
 
 ### Option Parameters
 
-Name | Type | Required | Description
-:--: | :--: | :------:| -----------
-server | String | Yes | The Vantiq server URL to connect to, e.g. `https://dev.vantiq.com`
-apiVersion | Integer | No | The version of the API to use.  Defaults to the latest.
+**`String server`**:  (*Required*) Specifies the VANTIQ server URL to connect to, e.g. `https://dev.vantiq.com` 
```

Comment:
I don't quite understand all these formatting changes. What's up with these?

Reply:
John Ellithorpe had this table style format for the docs that I really didn't like and I had a few hours spare so I changed it to what I felt was more readable. But I can revert to the original if that's preferred

Reply:
I kind of like the table format.  
What do you think, Jake?

Though I do like that you fixed several typos in the text :-)

Reply:
I think I prefer the table view as well, it does a better job of drawing your eyes to the parameters.